src/Sulu/Bundle/AdminBundle/Resources/public/js/vendor/husky/husky.css

```diff
@@ -10358,7 +10358,7 @@ a.btn-black, a.btn-highlight {
   display: inline-block;
   background: #52B6CA;
   padding: 0px 2px;
-  margin-right: 10px;
+  margin-right: 8px;
```

Comment:
i have commited this line accidentally in the develop (: https://github.com/massiveart/husky/commit/da2d4af8fe3ca03f4ec7baba2058e0df5ff01000 but its only this line (: